### DISCUSSION TITLE
SavvyCAN crashes Mac OSX 10.13.6 on any Menu->File-> selection

### DISCUSSION THREAD

**Discussion by TessieCatEC** [collin80/SavvyCAN#779] [Q&A]
I downloaded several times and reinstalled.  Yet SavvyCan is crashing my MacBook Air OSX 10.13.6 every time I select any of the menu items in the File -> dropdown menu.  Everything else in the program is working well to read CAN messages.  Any suggestions on how to correct this problem is appreciated.  I’ve attached the top portion of the Apple bomb message.  I am using SavvyCan to read (Wi-Fi) MrDIY CAN shield with an ESP32 that is loaded with ESP32RET.  That unit works great… it’s just SavvyCan app that is having fatal flaw. Collin, please help.
[SavvyCAN Error File brief.pdf](https://github.com/collin80/SavvyCAN/files/15276784/SavvyCAN.Error.File.brief.pdf)

**Comment by MrDIYca** [comment]
I am not sure if this is relevant, but 10.13 might be the issue; it is a bit outdated (2017)
Check out - https://github.com/collin80/SavvyCAN/discussions/760
